astrodb_utils/utils.py

```diff
@@ -205,15 +215,15 @@ def find_source_in_db(db, source, *, ra=None, dec=None, search_radius=60.0):
             )
             logger.debug(msg2)
             db_name_matches = db.query_region(
-                simbad_skycoord, radius=radius, ra_col="ra_deg", dec_col="dec_deg"
+                simbad_skycoord, radius=radius, ra_col=ra_col_name, dec_col=dec_col_name
```

Comment:
Consider putting this in a try/except.  You know that you get a `KeyError` if the column name is not correct.